src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,38 +1,33 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
-
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
     private Long id;
     private String content;
-    private List<Smile> smiles;
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
 
-    public String getContent() {
-        return content;
-    }
-
-    public void setContent(String content) {
-        this.content = content;
-    }
-
-    public List<Smile> getSmiles() {
-        return smiles;
-    }
-
-    public void setSmiles(List<Smile> smiles) {
-        this.smiles = smiles;
-    }
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
do we need to use FetchType.EAGER here?

Reply:
It works without Eager, but I know that OneToMany and ManyToMany default fetch types are Lazy. So I suggested it is better to get a message with smiles. But it works well without it, it is confusing me :(
I've removed it for the moment but need to ask on QnA ...